firestore-translate-text/functions/src/index.ts

```diff
@@ -79,8 +82,69 @@ export const fstranslate = functions.handler.firestore.document.onWrite(
     } catch (err) {
       logs.error(err);
     }
-  }
-);
+  });
+
+export const fstranslatebackfill = functions.tasks
+  .taskQueue()
+  .onDispatch(async (data: any) => {
+    const runtime = getExtensions().runtime();
+    if (!config.doBackfill) {
+      await runtime.setProcessingState(
+        "PROCESSING_COMPLETE",
+        "Existing documents were not backfilled."
```

Comment:
Might be better to say that "Existing documents were not translated because backfill was not requested". Maybe even add "If you would like to translate existing documents, reconfigure this extension and choose 'true' for 'Should existing, untranslated documents in the Firestore collection be translated as well?'"